oms-kemp-applicationdelivery-solution/azuredeploy.json

```diff
@@ -719,8 +717,8 @@
         {
             "type": "Microsoft.OperationsManagement/solutions",
             "name": "[variables('omsSolutions').customSolution.solutionName]",
-            "apiVersion": "[variables('omsLogAnalyticsAPIVersion')]",
-            "location": "[variables('location')]",
+            "apiVersion": "2015-11-01-preview",
+            "location": "[parameters('omsLogAnalyticsRegion')]",
             "dependsOn": [
                 "[resourceId('Microsoft.OperationalInsights/workspaces/', parameters('omsLogAnalyticsWorkspaceName'))]",
                 "[concat('Microsoft.OperationalInsights/workspaces/', parameters('omsLogAnalyticsWorkspaceName'), '/views/', variables('omsSolutions').customSolution.viewName)]",
```

Comment:
is ignorance a valid answer? :-) Transformed everything in resourceId (I like it more)